recipes/photospline/meta.yaml

```diff
@@ -1,22 +1,14 @@
-{% set name = "photospline" %}
-{% set org = "cnweaver" %}
-{% set version = "2.0.2" %}
-{% set file_ext = "tar.gz" %}
-{% set hash_type = "sha256" %}
-{% set hash_value = "0a3368205a7971a6919483ad5b5f0fbebb74614ec1891c95bb6a4fc9d3b950d4" %}
-
 package:
-  name: '{{ name|lower }}'
-  version: '{{ version }}'
+  name: photospline
+  version: 2.0.3
```

Comment:
Please use the jinja variable for the `version` and add it to the URL below so our bot can update the recipe automatically. (The rest of the jinja can be dropped like you did here b/c those are not helpful with anything.)